tests/foreman/api/test_host.py

```diff
@@ -126,43 +126,24 @@ def test_positive_get_per_page(self):
 
     @run_only_on('sat')
     @tier1
-    def test_positive_create_with_owner_type(self):
-        """Create a host and specify an ``owner_type``.
+    def test_negative_create_with_owner_type(self):
+        """Create a host and specify only ``owner_type``.
 
-        :id: 9f486875-1f30-4dcb-b7ce-b2cf515c413b
+        :id: cdf9d16f-1c47-498a-be48-901355385dde
 
-        :expectedresults: The host can be read back, and the ``owner_type``
-            attribute is correct.
+        :expectedresults: The host can't be created as ``owner`` is required.
 
         :CaseImportance: Critical
         """
         for owner_type in ('User', 'Usergroup'):
             with self.subTest(owner_type):
                 if owner_type == 'Usergroup' and bz_bug_is_open(1203865):
                     continue  # instead of skip for compatibility with py.test
-                host = entities.Host(owner_type=owner_type).create()
-                self.assertEqual(host.owner_type, owner_type)
-
-    @run_only_on('sat')
-    @tier1
-    def test_positive_update_owner_type(self):
-        """Update a host's ``owner_type``.
-
-        :id: b72cd8ef-3a0b-4d2d-94f9-9b64908d699a
-
-        :expectedresults: The host's ``owner_type`` attribute is updated as
-            requested.
-
-        :CaseImportance: Critical
-        """
-        host = entities.Host().create()
-        for owner_type in ('User', 'Usergroup'):
-            with self.subTest(owner_type):
-                if owner_type == 'Usergroup' and bz_bug_is_open(1210001):
-                    continue  # instead of skip for compatibility with py.test
-                host.owner_type = owner_type
-                host = host.update(['owner_type'])
-                self.assertEqual(host.owner_type, owner_type)
+                with self.assertRaises(HTTPError) as context:
+                    entities.Host(owner_type=owner_type).create()
+                self.assertEqual(context.exception.response.status_code, 422)
+                self.assertRegexpMatches(
+                    context.exception.response.text, "owner must be specified")
 
```

Comment:
As I said, `test_positive_create_with_user` and `test_positive_create_with_usergroup` completely covers this case:
* `test_positive_create_with_user` creates host with owner user and owner_type `user`
* `test_positive_create_with_usergroup` creates host with owner usergroup and owner_type `usergroup`
Other owner types do not exist.